pkg/deploy/service_accounts.go

```diff
@@ -49,43 +56,26 @@ func (s *ServiceAccounts) fetchServiceAccount(nsName string, saName string) (str
 		return "", fmt.Errorf("Internal inconsistency: Expected service account name to not be empty")
 	}
 
-	sa, err := s.coreClient.CoreV1().ServiceAccounts(nsName).Get(context.Background(), saName, metav1.GetOptions{})
+	tokenMgr := token.NewManager(s.coreClient, s.log)
+
+	t, err := tokenMgr.GetServiceAccountToken(nsName, saName, &authenticationv1.TokenRequest{
+		Spec: authenticationv1.TokenRequestSpec{
+			ExpirationSeconds: getInt64Point(5000), // 5 minutes
```

Comment:
a deploy might taken 20min to run as an example, i think our tokens needs to be longer e.g. 1 hour.

Reply:
nit: instead of having a method getInt64Point you can just set variable to 5000 and take a pointer of it.